taggit/admin.py

```diff
@@ -14,3 +19,66 @@ class TagAdmin(admin.ModelAdmin):
     ordering = ["name", "slug"]
     search_fields = ["name"]
     prepopulated_fields = {"slug": ["name"]}
+    actions = ["render_tag_form"]
+
+    def get_urls(self):
+        urls = super().get_urls()
+        custom_urls = [
+            path(
+                "merge-tags/",
+                self.admin_site.admin_view(self.merge_tags_view),
+                name="merge_tags",
+            ),
+        ]
+        return custom_urls + urls
+
+    @admin.action(description="Merge selected tags")
+    def render_tag_form(self, request, queryset):
+        selected = request.POST.getlist(admin.helpers.ACTION_CHECKBOX_NAME)
+        if not selected:
+            self.message_user(request, "Please select at least one tag.")
+            return redirect(request.get_full_path())
+
+        selected_tag_ids = ",".join(selected)
+        redirect_url = (
+            f"{request.get_full_path()}merge-tags/?selected_tags={selected_tag_ids}"
+        )
```

Comment:
instead of sending them over as GET params, I guess we could just from this point store the tag IDs in the session at this point and redirect over to the `merge_tags_view`? That way we avoid a serialization roundtrip (incidentally also avoid issues with GET param length as well)